baseapp_profiles/graphql/object_types.py

```
@@ -31,7 +31,17 @@ class BaseProfileUserRoleObjectType:
-        fields = ["id", "pk", "user", "role", "status"]
+        fields = [
+            "id",
+            "pk",
+            "user",
+            "role",
+            "status",
+            "invited_email",
+            "invited_at",
+            "invitation_expires_at",
+            "responded_at",
+        ]
```

Ruff (0.14.13)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)